src/hooks/useFetch.ts

```diff
@@ -1,39 +1,48 @@
 import { useState, useEffect } from 'react';
 import fetch from '@/helperFunctions/fetch';
 
-const useFetch = (url: string, options: object = {}) => {
+type EmptyFunction = () => void;
+
+const useFetch = (url: string, options: object = {}, isCall = true) => {
   const [response, setResponse] = useState<any>({});
```

Comment:
can we have a more explicit type here rather than `any`

Reply:
This change wasn't a part of my PR. can we please take it in another?